### PR TITLE
week 6

### PR DESCRIPTION
**Pull request by AmandaHockmuth**
added async wrappers and refactored, added error handling middleware and refactored

### CODE REVIEW

File: 03-task-manager/starter/routes/tasks.js [hunk 1]
```diff
@@ -6,9 +6,15 @@ const {
...
 } = require(`../controllers/tasks`);
 
 router.route("/").get(getAllTasks).post(createTask);
-router.route("/:id").get(getTask).patch(updateTask).delete(deleteTask);
+router
+  .route("/:id")
+  .get(getTask)
+  .patch(updateTask)
+  .delete(deleteTask)
+  // .put(replaceTask);
```
Comment (jsuoz121) [inline 03-task-manager/starter/routes/tasks.js:18]: fun, were you testing PUT methods?

File: 03-task-manager/starter/app.js [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+app.use(express.static(`./public`));
 app.use(express.json());
 
 //routes
-app.get("/hello", (req, res) => {
-  res.send("Task Manager App");
-});
 
 app.use(`/api/v1/tasks`, tasks);
+app.use(notFound);
```
Comment (jsuoz121) [inline 03-task-manager/starter/app.js:18]: Not a big deal either way but usually organization convention puts imports above any routes

**Review by jsuoz121 — APPROVED** [review]
Looks great!

File: 03-task-manager/starter/routes/tasks.js (hunk 1 above)
Comment (AmandaHockmuth) [reply 03-task-manager/starter/routes/tasks.js:18]: Yes! I was following along with the instructor and wanted to leave the PUT and some of the other sandbox code for future reference. Is that a poor practice for an entry level coder?

**Pull request merged**